coordinator/package.json

```diff
@@ -19,6 +19,7 @@
     "test:coverage": "jest --coverage",
     "types": "tsc -p tsconfig.json --noEmit",
     "download-zkeys:test": "ts-node ./scripts/downloadZkeys.ts test",
+    "download-zkeys:prod": "ts-node ./scripts/downloadZkeys.ts prod",
```

Comment:
Maybe it makes sense to move these commands to root package.json?

Reply:
can do, I guess the env variable will be overridden by the coordinator/.env file right?

Reply:
Yes, there is COORDINATOR_ZKEY_PATH in .env for coordinator